### PR TITLE
Fix ci build

### PR DESCRIPTION
**Pull request by bassemmagdy**

### CODE REVIEW

**Review by rg911 — COMMENTED** [review]
does this fixing the CI build on main?

File: src/components/AmountDisplay/AmountDisplayTs.ts [hunk 1]
```diff
@@ -42,7 +42,11 @@ export class AmountDisplayTs extends Vue {
 
     /// region computed properties getter/setter
     get integerPart(): string {
-        return this.value >= 0 ? Math.floor(this.value).toLocaleString() : Math.ceil(this.value).toLocaleString();
+        return this.value >= 0
```
Comment (yilmazbahadir) [inline src/components/AmountDisplay/AmountDisplayTs.ts:45]: Would this do the job?
```
return this.value >= 0
            ? Math.floor(this.value).toLocaleString() : '-' + Math.floor(this.value * -1).toLocaleString()
```

**Comment by bassemmagdy** [comment]
> does this fixing the CI build on main?

that should fix unit tests for ci-integration

File: src/components/AmountDisplay/AmountDisplayTs.ts (hunk 1 above)
Comment (bassemmagdy) [reply src/components/AmountDisplay/AmountDisplayTs.ts:45]: yes, the same thing but looks better!

**Review by yilmazbahadir — APPROVED** [review]

**Pull request merged**